src/pages/Reports/pages/OverviewMap/components/WebglMap.js

```diff
@@ -46,23 +48,40 @@ class WebglMap extends PureComponent {
       return;
     }
 
-    const { center, zoomIn, disabled, fitExtentOnPopupClose } = this.props;
+    const {
+      center,
+      zoomIn,
+      disabled,
+      fitExtentOnPopupClose,
+      isCTAButtonShifted
+    } = this.props;
+
+    const zoomTarget = config.reports.overviewMap.zoomDeepLinkedMarkers || 16;
 
     if (center) {
       const newCameraOptions = { center };
-      if (zoomIn) {
-        newCameraOptions.zoom =
-          config.reports.overviewMap.zoomDeepLinkedMarkers || 16;
+      if (isCTAButtonShifted) newCameraOptions.padding = { right: 400 };
+      if (zoomIn && this.map.getZoom() < zoomTarget) {
+        newCameraOptions.zoom = zoomTarget;
```

Comment:
I would like to see some comments here.
Besides that: nice and clean code 👍

Reply:
You are right, I will add some.